change_publishing_status_permission.module

```diff
@@ -32,6 +40,22 @@ function change_publishing_status_permission_form_node_form_alter(&$form, &$form
   }
 }
 
+/**
+ * Implements hook_node_access().
+ *
+ * Restricts editing and deleting of published nodes.
+ */
+function change_publishing_status_permission_node_access($node, $op, $account) {
+  $restricted_ops = ['update', 'delete'];
+  if (in_array($op, $restricted_ops) &&
```

Comment:
Please don’t wrap conditions: https://www.drupal.org/docs/develop/standards/coding-standards#linelength

Reply:
So instead have a very long line with conditions? Or spilt it over multiple ifs?